src/subcommand.rs

```diff
@@ -458,9 +494,19 @@ impl Subcommand {
         let mut line_width = UnicodeWidthStr::width(*name);
 
         for parameter in &recipe.parameters {
-          line_width += UnicodeWidthStr::width(
-            format!(" {}", parameter.color_display(Color::never())).as_str(),
-          );
+          line_width += if let Some(evaluator) = &mut evaluator {
+            UnicodeWidthStr::width(
+              format!(
+                " {}",
+                parameter
+                  .format_eval(evaluator)
+                  .color_display(Color::never())
```

Comment:
It should also be noted that we may want to keep the evaluated result here to avoid another evaluation later.